src/components/Home/Home.scss

```diff
@@ -91,18 +91,18 @@ body {
 .homepage--button {
   &--snackchat {
     width: 488px;
-    margin-top: 21px;
+    margin-top: 120px;
     background-color: $light-blue;
     color: $white;
     margin-bottom: 32px;
-    visibility: hidden; // Hidden for 1.0 release
   }
+
   &--nophoto {
-    background-color: $light-blue; // Colour changed for 1.0 release
+    background-color: $dark-blue;
     width: 488px;
     color: $white;
-    font-family: 'FaricyNew-Bold';
-    //border: 2px solid white; // Hidden for 1.0 release
+    font-family: 'FaricyNew';
+    border: 2px solid white !important;
```

Comment:
it's not ideal to be using !important 
do you really need it?

Reply:
I tried to avoid using it, used it as a quick fix because the rest of the styles have changed since this was originally commented out. I could spend some time refactoring the styles to sort this if you think it is worth it?

Reply:
Fixed :)